packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -66,21 +66,31 @@ export async function createNewFluidFile(
 
     let itemId: string;
     let summaryHandle: string = "";
-
+    let sharingLink: string = "";
+    let sharingLinkErrorReason: string = "";
     if (createNewSummary === undefined) {
         itemId = await createNewEmptyFluidFile(getStorageToken, newFileInfo, logger, epochTracker);
     } else {
         const content = await createNewFluidFileFromSummary(
             getStorageToken, newFileInfo, logger, createNewSummary, epochTracker);
         itemId = content.itemId;
         summaryHandle = content.id;
+        sharingLink = content.sharingLink || "";
+        sharingLinkErrorReason = content.sharingLinkErrorReason || "";
     }
 
     const odspUrl = createOdspUrl({... newFileInfo, itemId, dataStorePath: "/"});
     const resolver = new OdspDriverUrlResolver();
     const odspResolvedUrl = await resolver.resolve({ url: odspUrl });
-    fileEntry.docId = odspResolvedUrl.hashedDocumentId;
-    fileEntry.resolvedUrl = odspResolvedUrl;
```

Comment:
I removed the above two lines as they were modifying an input parameter which then was not consumed anywhere.

Reply:
It is used for caching purposes in epoch tracker. We don't have this info before creating the file in this container, so it needs to be added.